lib/explorer/data_frame.ex

```diff
@@ -441,18 +441,42 @@ defmodule Explorer.DataFrame do
   ## Examples
 
       iex> df = Explorer.DataFrame.from_columns(floats: [1.0, 2.0], ints: [1, nil])
-      iex> Explorer.DataFrame.to_map(df)
+      iex> Explorer.DataFrame.to_columns(df)
       %{"floats" => [1.0, 2.0], "ints" => [1, nil]}
 
       iex> df = Explorer.DataFrame.from_columns(floats: [1.0, 2.0], ints: [1, nil])
-      iex> Explorer.DataFrame.to_map(df, atom_keys: true)
+      iex> Explorer.DataFrame.to_columns(df, atom_keys: true)
       %{floats: [1.0, 2.0], ints: [1, nil]}
   """
-  @spec to_map(df :: DataFrame.t(), Keyword.t()) :: map()
-  def to_map(df, opts \\ []) do
+  @spec to_columns(df :: DataFrame.t(), Keyword.t()) :: map()
+  def to_columns(df, opts \\ []) do
     opts = Keyword.validate!(opts, convert_series: true, atom_keys: false)
 
-    apply_impl(df, :to_map, [opts[:convert_series], opts[:atom_keys]])
+    apply_impl(df, :to_columns, [opts[:convert_series], opts[:atom_keys]])
+  end
+
+  @doc """
+  Converts a dataframe to a list of maps (rows).
+
```

Comment:
Perhaps it is worth adding a disclaimer that this is potentially an expensive operation because Explorer often keeps data in a columnar format?